packages/backend/src/database/seeds/development/01_initial_user.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+import bcrypt from 'bcrypt';
+import {
+    SEED_EMAIL,
+    SEED_ORGANIZATION,
+    SEED_ORGANIZATION_MEMBERSHIP,
+    SEED_PASSWORD,
+    SEED_USER,
+} from 'common';
+
+export async function seed(knex: Knex): Promise<void> {
+    // Deletes ALL existing entries
+    await knex('users').del();
+    await knex('organizations').del();
```

Comment:
Should we also clear emails, organization_memberships and password_logins tables ? 
If so, we should also clear all the tables in the other seed scripts.

Reply:
These get dropped with `CASCADE` after the user. Would you prefer it was manual?